cookiecutter/{{ cookiecutter.package_name }}/setup.py

```diff
@@ -0,0 +1,27 @@
+from pathlib import Path
+
+import setuptools
+
+this_directory = Path(__file__).parent
+long_description = (this_directory / "README.md").read_text()
+
+setuptools.setup(
+    name="{{ cookiecutter.package_name }}",
+    version="0.0.1",
+    author="{{ cookiecutter.author_name }}",
+    author_email="{{ cookiecutter.author_email }}",
+    description="{{ cookiecutter.description }}",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="",
+    packages=setuptools.find_packages(),
+    include_package_data=True,
+    classifiers=[],
+    python_requires=">=3.7",
+    install_requires=[
+        # By definition, a Custom Component depends on Streamlit.
+        # If your component has other Python dependencies, list
+        # them here.
+        "streamlit >= 0.63",
```

Comment:
why such an old version ? wouldn't >1.0.0 be enough ?

Reply:
This is defined in the current templates, and this PR is based on the existing ones, so if we need to change it, we should change it as a separate PR.